### ISSUE TITLE
(Optional) Example flags

### ISSUE THREAD

**Issue by mhesselbarth** [equitable-equations/fqar#12]
https://github.com/openjournals/joss-reviews/issues/6366

This one is optional, but make sure you actually want to use `\donttest{}` and not `\dontrun{}`. As far as I understand it, the first only doesn't run tests on CRAN, while the later really doesn't run the examples. In my opinion, `\dontrun{}` is much more common, which is why I was a bit confused by the `\donttest{}`

**Comment by equitable-equations** [comment]
Yeah, this is the result of specific instructions from the initial human CRAN reviewer who didn't like the use of `dontrun`, possibly because I'd applied it to *all* downloading functions at that time. Honestly I just did what CRAN said.

The package has since been updated to be more stable under internet interruptions, so this can be probably be changed without causing them too much frustration. I agree it would be a positive update and I'll explore it.
